src/components/shared/footer.tsx

```diff
@@ -0,0 +1,100 @@
+import { css } from '@emotion/core';
+import { Footer } from '@guardian/src-footer';
+import { brandBorder, brandText, space } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+import { bool } from 'aws-sdk/clients/signer';
+import React from 'react';
+
+
+// Footer content styles 
+const container = css`
+		border-style: solid;
+		border-color: ${brandBorder.primary};
+		border-width: 0 1px;
+	
+`
+
+const para = css`
+	${textSans.medium({ lineHeight: "tight" })};
+	max-width: 58.75rem;
+`
+
+const ul = css`
+	list-style: none;
+	padding: 0;
+	border-top: 1px solid ${brandBorder.primary};
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-template-areas:
+		"link1 link3"
+		"link2 link4";
+`
+
+const li = css`
+	padding: ${space[1]}px 0 ${space[4]}px 0;
+	border-style: solid;
+	border-color: ${brandBorder.primary};
+	border-width: 0;
+`
+const anchor = css`
+	${textSans.medium({ lineHeight: "regular" })};
+	color: ${brandText.anchorPrimary};
+	text-decoration: none;
+`
+
+const link1 = css`
+	grid-area: link1;
+	border-right-width: 1px;
+`
+const link2 = css`
+	grid-area: link2;
+	border-right-width: 1px;
+`
+
+const CcpaListItem = (ccpaStatus: bool) => {
+    if (ccpaStatus){
+        return (
+            <li css={[li, link2]}>
+				<a  id='testing' css={anchor} href="https://www.theguardian.com/ccpaoptedin">
```

Comment:
Can remove the id here. Do we need confirmed text and ink here from Product?

Reply:
Yes, I think we wanted a placeholder, Ill confirm with @maxspencer - I just placed a non-existent URL for Max to find